Tests/ObjectMapperTests/MappableExtensionsTests.swift

```diff
@@ -57,6 +57,60 @@ func ==(lhs: TestMappable, rhs: TestMappable) -> Bool {
 	return lhs.value == rhs.value
 }
 
+class CustomDictionary<KeyType: Hashable, ValueType>: Mappable {
```

Comment:
Below you are passing in a `Mappable` type for `ValueType`. If you expect `ValueType` to map, it needs to conform to `Mappable` in the class definition. 

Not 100% sure, but this might fix your issue:
`class CustomDictionary<KeyType: Hashable, ValueType: Mappable>: Mappable`

Reply:
No luck :(

Reply:
The debugger doesn't like all the custom operators but I added a bunch of NSLogs and finally figured out the issue:
```
public func <- <T: BaseMappable>(left: inout Dictionary<String, T>, right: Map) {
```
My custom map doesn't specify that the keys have to be `String`s, just `Hashable`s.  Which means the Dictionary operator doesn't get selected, and instead it uses the "Object of Basic type" one and that one doesn't transform the `TestMappable`s to json, which makes `isValidJSONObject` return false and `toJSONString` ends up returning nil.

Is this just a "make CustomDictionary have Strings for keys" situation?  I can't really think of a situation where non-String keys would make sense.